pkg/pillar/cmd/zedagent/attesttask.go

```diff
@@ -300,7 +313,76 @@ func (server *VerifierImpl) SendAttestQuote(ctx *zattest.Context) error {
 
 //SendAttestEscrow implements SendAttestEscrow method of zattest.Verifier
 func (server *VerifierImpl) SendAttestEscrow(ctx *zattest.Context) error {
-	//XXX: Fill it in when Controller code is ready
+	if ctx.OpaqueCtx == nil {
+		log.Fatalf("[ATTEST] Uninitialized access to OpaqueCtx")
+	}
+	attestCtx, ok := ctx.OpaqueCtx.(*attestContext)
+	if !ok {
+		log.Fatalf("[ATTEST] Unexpected type from opaque ctx: %T",
+			ctx.OpaqueCtx)
+	}
+	// bail if V2API is not supported
+	if !zedcloud.UseV2API() {
+		return zattest.ErrControllerReqFailed
+	}
```

Comment:
What happens with this error? Would it make sense to have a separate “not supported” error which wouldn’t be logged as an error?

Reply:
I've introduced a new error code "ErrNoVerifier" to indicate that Controller does not support Verifier functionality. And changed these places to return ErrNoVerifier instead of ErrControllerReqFailed. On ErrNoVerifier, the state machine does not retry sending the same request again, and waits on the respective state (potentially forever, unless there is an explicit request from zedagent to restart attestation)

Reply:
ok